docs/usage/help.rst.inc

```diff
@@ -55,7 +55,8 @@ Shell-style patterns, selector ``sh:``
     Like fnmatch patterns these are similar to shell patterns. The difference
     is that the pattern may include ``**/`` for matching zero or more directory
     levels, ``*`` for matching zero or more arbitrary characters with the
-    exception of any path separator. A leading path separator is always removed.
+    exception of any path separator, ``{}`` containing comma-separated alternative
+    patterns. A leading path separator is always removed.
```

Comment:
sorry, we need another change, missed that in my previous reviews:

see the comment in line 1! you need to locate this text in the code, likely in `src/borg/archiver/...` and change it (only) there.

Reply:
Ah, very cool. I have moved the text. As I understand the development documentation, I don't need to do anything further, as the maintainers prefer to build the docs on a per-release basis, correct?